docs/content/release_notes/changelog_1.9.0.md

```diff
@@ -0,0 +1,8 @@
+# 1.9.0 (March 2022)
+
+IBM® block storage CSI driver 1.9.0 adds new support and enhancements:
+- New stretched topology support for SAN Volume Controller storage systems
+- New Call Home support for Spectrum Virtualize storage systems
+- Now enables I/O group configuration
```

Comment:
```suggestion
- Now enables I/O group configuration for Spectrum Virtualize family storage systems
```
should we mention it also in `creating_volumestorageclass.md`? I don't see note about it there

Reply:
what type of information needs to be added there? I was told to add it to the what's new

Reply:
update on this file - done

Reply:
I think we can note that this parameter is for SpecV family only in the storageclass page as well

Reply:
done - also added to limitations

Reply:
I spoke to @erantzabari  and we should mark it as a limitation. I'm closing this 1.9.0 documentation now. Please open a ticket for me so that in the next update we can add in the SpaceEfficiency limitations as well (and anything else that might have been missed)

Reply:
I don't think a lack of parameter is a limitation, I think the fact that a9k doesn't burden the user with low-level details such as space efficiency is by design and makes the a9k space efficiency feature great